ironfish-cli/src/flags.ts

```diff
@@ -111,14 +113,23 @@ export const IronFlag = Flags.custom<bigint, IronOpts>({
   char: 'i',
 })
 
-const parseIron = (input: string, opts: IronOpts): Promise<bigint> => {
+export const parseIron = (input: string, opts: IronOpts): Promise<bigint> => {
   return new Promise((resolve, reject) => {
     const { largerThan, flagName } = opts ?? {}
-    const value = CurrencyUtils.decodeIron(input)
-    if (largerThan !== undefined && value <= largerThan) {
-      reject(new Error(`The minimum ${flagName} is ${CurrencyUtils.renderOre(1n, true)}`))
-    }
+    try {
+      const value = CurrencyUtils.decodeIron(input)
+
+      if (largerThan !== undefined && value <= largerThan) {
+        reject(new Error(`The minimum ${flagName} is ${CurrencyUtils.renderOre(1n, true)}`))
```

Comment:
nit - I just noticed this in the existing error message -- it should use `largerThan` instead of `1n` for the minimum amount